android/vendored/unversioned/@shopify/react-native-skia/cpp/api/JsiSkDataFactory.h

```diff
@@ -64,15 +64,15 @@ namespace RNSkia {
             size_t len;
             auto err = SkBase64::Decode(&base64.utf8(runtime).c_str()[0], size, nullptr, &len);
             if(err != SkBase64::Error::kNoError) {
-              jsi::detail::throwJSError(runtime, "Error decoding base64 string");
+              throw jsi::JSError(runtime, "Error decoding base64 string");
               return jsi::Value::undefined();
```

Comment:
Nitpick: since it throws now, the `return` is not needed 😄

Reply:
i don't want to change too much code for skia, the latest skia already fixed the issue. we will update the vendored modules soon.